docs/Dashboard/user_guide.md

```diff
@@ -15,13 +15,10 @@ It has three components.
 - *Saved Runs* lists the saved runs which can be published to share with others
 - *Expiring Runs* lists the saved runs which will be deleted from the server within the next 20 days
 
-
 ![Overview](assets/images/Overview.png)
 
-
-The *User Profile* page can be used to view profile information from the OIDC authentication as well as to view and manage Pbench Server API keys. This page is accessed by selecting the *My profile* option from the dropdown menu activated by clicking on the username at the right end of the header bar. 
+The *User Profile* page can be used to view profile information from the OIDC authentication as well as to view and manage Pbench Server API keys. This page is accessed by selecting the *My profile* option from the dropdown menu activated by clicking on the username at the right end of the header bar.
 
```

Comment:
> \*User Profile*

You used `_User Profile_` in the `README.md`...in case you want to be consistent.